web-dev/src/components/NavigationTopBar/index.js

```diff
@@ -37,7 +37,7 @@ const NavigationTopBar = () => {
             {/* Search button */}
             <li className="nav-item">
               <NavLink className="nav-link" to="/search">
```

Comment:
This should prevent underline on the search text
```suggestion
              <NavLink className="nav-link text-decoration-none" to="/search">
```